azuread/resource_group.go

```diff
@@ -44,6 +44,7 @@ func resourceGroup() *schema.Resource {
 			"members": {
 				Type:     schema.TypeSet,
 				Optional: true,
+				Computed: true,
```

Comment:
What's the rationale behind this field being computed, @evenh ? To me, this sounds strange?

Reply:
@landro, this is because members can also be managed via the separate resource or cli or azure. Having it be computed allows the resource to read them in if unset. and then if set try and force them to be X.